.github/ISSUE_TEMPLATE/question-query-template.md

```diff
@@ -14,9 +14,9 @@ A clear and concise ask/query.
 A clear explanation of why is this not a bug or a feature request?
 
 **Setup (please complete the following information if applicable):**
- - OS: [e.g. iOS]
- - IDE : [e.g. IntelliJ]
- - Version of the Library used
+ - Name and Version of the Library package used: [e.g. Azure.Storage.Blobs 12.2.0] 
+ - OS and .NET runtime version: [e.g. Linux dotnet core 3.1]
```

Comment:
When we say "clear and concise" above it sounds a bit patronizing.  How about something like "How can we help?"  or "What is your question?"  and assume people will provide the necessary details and if not we can ask follow up questions.

Reply:
Same goes for the other templates.

Reply:
Agreed but unrelated to the change.